src/features/diagnosticsProvider.ts

```diff
@@ -122,7 +122,7 @@ class DiagnosticsProvider extends AbstractSupport {
     private _disposable: CompositeDisposable;
     private _diagnostics: vscode.DiagnosticCollection;
     private _validateCurrentDocumentPipe = new Subject<vscode.TextDocument>();
-    private _validateAllPipe = new Subject();
+    private _validateAllPipe = new Subject<string>();
```

Comment:
This was useful for debugging to see what event was causing all diagnostics to be requested.